queries.sql

```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
-
-SELECT * from animals WHERE name = 'Luna';
+SELECT * from animals;
+SELECT * from animals where name like '%mon';
+SELECT name from animals WHERE date_of_birth BETWEEN '2016-01-01' and '2019-01-01';
```

Comment:
- [ ] For this requirement, ` list the name of all animals born between 2016 and 2019.` Here you should list the name of all animals starting from the first day of the first month of 2016 up to the last day of the last month of 2019, hope this makes sense 🤔 Your code might look like this:

```sql
SELECT name FROM animals WHERE date_of_birth >= '2016/01/01' AND date_of_birth <= '2019/31/12';
# 0r 
SELECT name FROM animals WHERE EXTRACT(year FROM date_of_birth) BETWEEN 2016 AND 2019;
```


- Please correct this 👍🏻